sdk-core/src/main/java/care/data4life/sdk/tag/TaggingContract.kt

```diff
@@ -51,16 +50,22 @@ class TaggingContract {
             encryptionKey: GCKey,
             prefix: String = ""
         ): MutableList<String>
+
+        companion object {
+            internal val IV = ByteArray(16)
+        }
     }
 
-    // TODO: make this package internal
-    interface Helper {
-        fun convertToTagMap(tagList: List<String>): Tags
+    interface Converter {
+        @Throws(D4LException::class)
+        fun toTags(tagList: List<String>): Tags
+    }
 
+    // TODO: make this package internal
```

Comment:
Encoding is intended to be package internal, but since it used somewhere else, it cannot be right now...so I'd like to keep it. Are you okay with that?

Reply:
Yes, it's fine 👍